### DISCUSSION TITLE
Not correctly stopping playlist

### DISCUSSION THREAD

**Discussion by AVK-RTX** [savonet/liquidsoap#4117] [General]
Hello, 
I use the LiquidSoap 2.2.5 build. 

use a playlist like:
air_adv_pl = playlist (loop=false, reload=360, mode="normal", "/radio_base/lr-rezerv_playlist_local/lr_test.pl") # mp3 file playlist 
radio_stream_in_ext = input.external.rawaudio(..) # rtp input stream decoder

start_ad_air = link (false)
start_ad_air = active.bool('start_ad_air', false)

start_gst_air = ref(true)
start_gst_air = active.bool('start_gst_air', true)

radio_stream_out_ext = switch (track_sensitivity = false, [(start_ad_air, air_adv_pl), (start_gst_air, radio_stream_in_ext)])

when you start the playlist air_adv_pl (start_ad_air = true), the playlist starts everything is ok, 
when I stop air_adv_pl (start_ad_air = false) - the switch remembers the state of the activated track (pauses)
AND WHEN air_adv_pl is restarted (start_ad_air = true), the play starts from the same file (location) where the stop was.

How to get around this situation and run air_adv_pl from the beginning every time.
How to make a playlist play from the beginning every time (start_ad_air = true)?

Thank you!

**Comment by pavelzabavin** [comment]
Good question, I hope the developers will help.

**Comment by toots** [comment]
You shoud call `playlist.reload()` when you want the playlist to restart!

**Comment by pavelzabavin** [comment]
@toots I randomly determined that the file can be unpaused using the 'playlist.skip' command, and then 'playlist.reload'. This will stop the playlist and return to the first position. I would just like to see the 'playlist.stop' function.

**Comment by pavelzabavin** [comment]
How to make a complete stop of a playlist correctly??
